### PR TITLE
PR of the day, a day late | I HATE THE ANTICHRIST!! | Upgrades the SolFed response outfits (except the engineering one)

### PR DESCRIPTION
**Pull request by Paxilmaniac**
## About The Pull Request

As the title says, the SolFed emergency call outfits have been updated with some of the new stuff we have, and some new items I'm adding for them.

image

Marshalls no longer look like wild west larpers, instead wearing a solfed-blue flak jacket alongside their uniform.

image

SWAT is now some proper SolFed peacekeepers, sporting blue helmets and all.

image

The military is now no longer just dressed as the HECU, and no longer uses the m16 (amen).

Note that the gas mask is the wrong one here, I just forgot to set it to the right one.

image

EMTs are now equipped like some real deal galactic EMTs, instead of just station paramedics

Of course, the new clothing items can also be found in cargo, because I'm fun like that.

## How This Contributes To The Skyrat Roleplay Experience

It's hard to take the SolFed call guys seriously when they are all either larping as wild west cowboys, or when they're dressed like the fucking hecu from half life. Now that we have actual thematic armor and clothing for SolFed, its only right that their response teams are dressed with them.

## Proof of Testing

See screenshots above for more

image
image

## Changelog

:cl:
balance: SolFed response team equipment has been changed around a bit with the new armor and whatnot.
image: SolFed response teams (and cargo imports) can now see a new set of SolFed uniforms, one for peacekeepers and one for EMTs. No longer do they have to dress as wild west larpers, or half life HECU.
/:cl:

### CODE REVIEW

**Comment by projectkepler-ru** [comment]
Finally

I can really hate the anti christ

File: modular_skyrat/modules/goofsec/code/sol_fed.dm [hunk 1]
```diff
@@ -321,21 +321,24 @@ GLOBAL_LIST_INIT(call911_do_and_do_not, list(
...
 	l_pocket = /obj/item/restraints/handcuffs
 	id = /obj/item/card/id/advanced/solfed
-	backpack_contents = list(/obj/item/storage/box/survival = 1,
+	backpack_contents = list(
+		/obj/item/storage/box/survival = 1,
 		/obj/item/storage/box/handcuffs = 1,
-		/obj/item/melee/baton/security/loaded = 1,
 		/obj/item/solfed_reporter/swat_caller = 1,
-		/obj/item/beamout_tool = 1)
+		/obj/item/beamout_tool = 1
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/goofsec/code/sol_fed.dm]:
```suggestion
		/obj/item/beamout_tool = 1,
```

File: modular_skyrat/modules/goofsec/code/sol_fed.dm [hunk 2]
```diff
@@ -482,22 +491,25 @@ GLOBAL_LIST_INIT(call911_do_and_do_not, list(
...
 	id = /obj/item/card/id/advanced/solfed
-	r_hand = /obj/item/gun/ballistic/automatic/m16
-	backpack_contents = list(/obj/item/storage/box/handcuffs = 1,
+	r_hand = /obj/item/gun/ballistic/automatic/sol_rifle
+	backpack_contents = list(
+		/obj/item/storage/box/handcuffs = 1,
+		/obj/item/sacrificial_face_shield = 1,
 		/obj/item/melee/baton/security/loaded = 1,
-		/obj/item/ammo_box/magazine/m16 = 4
+		/obj/item/ammo_box/magazine/c40sol_rifle/standard = 4
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/goofsec/code/sol_fed.dm]:
```suggestion
		/obj/item/ammo_box/magazine/c40sol_rifle/standard = 4,
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Tiny nitpicks that I'll commit myself.

**Review by GoldenAlpharex — APPROVED** [review]
Okay, this should be good to go now.

**Pull request merged**